.github/workflows/assign_issue.yml

```diff
@@ -0,0 +1,23 @@
+---
+name: "Assign issue to a team"
+
+on:
+    issues:
+        types: [opened, reopened]
+
+jobs:
+  auto_assign_issue:
+    runs-on: ubuntu-latest
+    container: ghcr.io/datadog/agent-issue-auto-assign:latest
```

Comment:
The definition of the container is [here](https://github.com/DataDog/datadog-agent-models/blob/nschweitzer/publish/Dockerfile), and the model generation code is [here](https://github.com/DataDog/datadog-agent/pull/26207/files#diff-2ca0cfe766f380fda89637f61527d0e328eda1924c480bbe0077693bf7081004R8)